package.json

```diff
@@ -126,6 +126,7 @@
     "protobufjs": "~6.8.6",
     "ps-node": "^0.1.6",
     "recast": "^0.17.3",
+    "request": "^2.88.0",
```

Comment:
Upgrading `karma-sauce-launcher` removed `request` module which is required by "request-promise-native". So I had to re-install this again.